synapse/lib/nexus.py

```diff
@@ -203,6 +206,18 @@ async def index(self):
         else:
             return self.nexshot.get('nexs:indx')
 
+    async def setindex(self, indx):
+
+        nexsindx = await self.index()
+        if indx < nexsindx:
+            mesg = f'setindex ({indx}) is less than current index ({nexsindx})'
```

Comment:
Probably doesnt effect our use case, but we may wanna consider returning a bool rather than raising here. We could end up crippling a cortex if it somehow ends up trying to set the index backward.

Reply:
Yeah I wasn't sure if it should raise or return False, I can change it